doc/source/science_guide/sg_dynamics.rst

```diff
@@ -236,8 +236,31 @@ pending further testing.
 Seabed stress
 ***************
 
-The parameterization for the seabed stress is described in :cite:`Lemieux16`. The components of the basal seabed stress are 
-:math:`\tau_{bx}=C_bu` and :math:`\tau_{by}=C_bv`, where :math:`C_b` is a coefficient expressed as
+CICE now includes two options for calculating the seabed stress,
```

Comment:
minor: I  would not use words such as  "now"  (or "new", a little bit below), since it is not clear to the reader when is "now". The documentation reflects the actual state of the code, so just "CICE includes two options..." would be better in my opinion.